### PR TITLE
2 for 1: configuration markdown & usage sample

### PR DESCRIPTION
**Pull request by Amir974**
Fixing configuration markdown #6385 (Easyfix)
also, added a simple switch example for AC management.

## Checklist:

- [x] Branch: `current` 
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/climate.sensibo.markdown [hunk 1]
```diff
@@ -17,12 +17,14 @@ Integrates [Sensibo](https://sensibo.com) Air Conditioning controller into Home
...
 To enable this platform, add the following lines to your `configuration.yaml` file:
 
+{% configuration %}
 ```yaml
 # Example configuration.yaml entry
 climate:
   - platform: sensibo
     api_key: <your_key_here>
 ```
+{% endconfiguration %}
```
Comment (fabaff) [inline source/_components/climate.sensibo.markdown]: `{% configuration %}` is for the [variables](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration) and not the sample itself.

**Review by frenck — CHANGES_REQUESTED** [review]
@Amir974 Can you take a look at the comments? 👍

File: source/_components/climate.sensibo.markdown (hunk 1 above)
Comment (Amir974) [reply source/_components/climate.sensibo.markdown]: On it, sorry for the misunderstanding

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thank you for updating the PR so quickly @Amir974 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**